test/jdk/sun/security/pkcs11/PKCS11Test.java

```diff
@@ -931,9 +942,9 @@ protected void copyNssCertKeyToClassesDir(Path dbPath) throws IOException {
     public static enum ECCState {None, Basic, Extended}
 
     @Artifact(
-            organization = "jpg.tests.jdk.nsslib",
+            organization = NSSLIB,
             name = "nsslib-windows_x64",
-            revision = "3.46-VS2017",
```

Comment:
nsslib-windows_x86 support is not updated to 3.91. Is it possible to upgrade this (although we don't have it as a supported platform)

Reply:
I spent a few hours trying to get this to compile but the build environment is corrupted somehow and didn't want to spend a lot of time on this since it's not supported. I can make a new bug to support if if we need it.